manifests/generated/rbac-operator.authorization.k8s.yaml.in

```diff
@@ -170,6 +187,17 @@ rules:
   verbs:
   - watch
   - list
+- apiGroups:
+  - certificates.k8s.io
+  resources:
+  - certificatesigningrequests
+  verbs:
+  - create
+  - update
+  - get
+  - delete
```

Comment:
do we delete requests? i thought k8s garbage colleceted them for us. 

Also, this rule looks like a duplicate. there's already an entry for ```certificatesigningrequests```